service/src/main/kotlin/io/provenance/api/frameworks/web/config/WebConfig.kt

```diff
@@ -4,17 +4,29 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import org.springframework.context.annotation.Configuration
 import org.springframework.http.codec.ServerCodecConfigurer
 import org.springframework.http.codec.json.Jackson2JsonEncoder
+import org.springframework.http.codec.multipart.DefaultPartHttpMessageReader
+import org.springframework.http.codec.multipart.MultipartHttpMessageReader
 import org.springframework.web.reactive.config.EnableWebFlux
 import org.springframework.web.reactive.config.WebFluxConfigurer
 
 @Configuration
 @EnableWebFlux
 class WebConfig(
-    private val objectMapper: ObjectMapper
+    private val objectMapper: ObjectMapper,
 ) : WebFluxConfigurer {
 
     override fun configureHttpMessageCodecs(configurer: ServerCodecConfigurer) {
         // Enable support for serializing protos to JSON in API responses
         configurer.defaultCodecs().jackson2JsonEncoder(Jackson2JsonEncoder(objectMapper))
+        configurer.defaultCodecs().maxInMemorySize(500 * 1024 * 1024)
```

Comment:
previously, i would have expected this value to stay small such that we wrote to disk more often. I do see this value is being set in figure services with comments like `Set max in memory size to 5 MB to accommodate large loan data` so giving this a go.